src/bin/nydus-image/main.rs

```diff
@@ -1479,6 +1494,31 @@ impl Command {
         }
     }
 
+    fn get_blob_cache_storage(
+        matches: &ArgMatches,
+        conversion_type: ConversionType,
+    ) -> Result<Option<ArtifactStorage>> {
+        if let Some(p) = matches.get_one::<PathBuf>("blob-cache-dir") {
+            if conversion_type == ConversionType::TarToTarfs
```

Comment:
How about using whitelist instead of blacklist here?

Reply:
There are more whitelists than Blacklist, we support xxxToRef and xxxToRafs